modules/renderer/features.js

```diff
@@ -392,13 +401,13 @@ export function rendererFeatures(context) {
             var hasMatch = false;
 
             for (var i = 0; i < _keys.length; i++) {
+                const parents = features.getParents(entity, resolver, geometry);
```

Comment:
we should move this line outside of the `for` loop.

Reply:
I've made these two changes